src/utils.ts

```diff
@@ -101,11 +101,44 @@ export function getMedicationsByConceptEndpoint(conceptUuid: string) {
 }
 
 /**
- * Given an array of CodeableConcept condings, return the first one without an associated system (which should be the uuid of the underyling concept)
+ * Given an array of CodeableConcept codings, return the first one without an associated system (which should be the concept-referenced-by-uuid coding)
  * @param codings
  */
-export function getConceptUuidCoding(codings: Coding[]) {
-  return codings ? codings.find((c) => !("system" in c))?.code : null;
+export function getConceptCoding(codings: Coding[]) {
+  return codings ? codings.find((c) => !("system" in c)) : null;
+}
+
+/**
+ * Given an array of CodeableConcept codings, return the code for the first one without an associated system (which should be the concept-referenced-by-uuid coding)
+ * @param codings
+ */
+export function getConceptCodingUuid(codings: Coding[]) {
+  return getConceptCoding(codings)?.code;
+}
+
+/**
+ * Given an array of CodeableConcept codings, return the display for the first one without an associated system (which should be the concept-referenced-by-uuid coding)
+ * @param codings
+ */
+export function getConceptCodingDisplay(codings: Coding[]) {
+  return getConceptCoding(codings)?.display;
+}
+
+/**
```

Comment:
This new function is the  heart of this PR...  I've extracted the display of a medication out to this method and tweaked the display as described below.